governance/internationals.md

```diff
@@ -0,0 +1,55 @@
+# International Bible Quizzing
+These rules apply to the Internationals quiz meet conducted once yearly, administered under the oversight of the CQLT.
+
+## Eligibility Requirements
+- All quizzers for International competition must be 12 to 18 years of age by the meet registration date or have been enrolled as a full-time high school student during the current quiz year.
+- In order to quiz at Internationals for a specific district, the quizzer must reside in that district or in a bordering community.  Their home church must be from the district in which they quiz.  Any exceptions to this must be brought to the C&MA Quizzing Leadership Team (CQLT).
+
+## Team Structure
+- Each district will be allowed to send as many teams as the district feels will be competitive and that they are financially able to send.  These will be the district International Teams.
+- A team will be composed of a maximum of five members: four starting quizzers and one substitute.  These five shall constitute the team for International Finals.  Any exceptions to this must be brought to the C&MA Quizzing Leadership Team (CQLT).
```

Comment:
Should this be here or in the rule book proper?

Reply:
This is under roles and responsibilities already "Teams can have up to 5 quizzers on it. Teams can have up to 4 quizzers in the quiz at any given time."

So might this be redundant under Internationals?  And technically it's not 100% accurate...Internationals teams are allowed to be composed of < 5 members. But this rule states exactly 5 is the only valid config.

Reply:
Yeah, I say delete it here. IBQ can inherit what's in the rule book.